lib/screens/authentication/recover/recover_account_found/interactor/mappers/fetch_recover_guardian_state_mapper.dart

```diff
@@ -50,7 +50,7 @@ class FetchRecoverRecoveryStateMapper extends StateMapper {
       }
 
       // Save the private key and account
```

Comment:
This is redundant - the account and key have already been saved when the recovery was initiated.

Recovery process is pretty confusing. 

Anyway, I am not changing this code, it was saving these two items before and now still saving them. 

But - we don't need this and it is a bit of a risk the state got out of sync.

Recovery process should be fixed up on a future PR.

Reply:
Yeah I also think that when I saw that saving instruction.